### ISSUE TITLE
Update experiment template to use InternalInterface

### ISSUE THREAD

**Issue by relet** [MONROE-PROJECT/Experiments#18]
Afaics the experiment template still uses InterfaceName instead of InternalInterface metadata.

Quoth Eneko:
My second question is related to a test that has been completed "correctly" or at least "Finished" for the system. This test has been even easier, removing my first entrypoint and directly executing the python script responsible for gathering the metadata. Even though the experiment seems OK, when I read the output files, there is no metadata available. In fact, the only thing is an error related to this:

```
2016-08-08T11:33:03.130232163Z Trying to get metadata for Telenor SE
2016-08-08T11:33:03.130255470Z Trying to get metadata for Telenor SE
2016-08-08T11:33:03.130278068Z Trying to get metadata for Telenor SE
2016-08-08T11:33:03.130300597Z Trying to get metadata for Telenor SE
2016-08-08T11:33:03.130419780Z Traceback (most recent call last):
2016-08-08T11:33:03.130448126Z   File "/opt/monroe/experiment.py", line 282, in <module>
2016-08-08T11:33:03.130479230Z     ifname = meta_info['modem']['InterfaceName']
2016-08-08T11:33:03.130502398Z   File "<string>", line 2, in __getitem__
2016-08-08T11:33:03.130526858Z   File "/usr/lib/python2.7/multiprocessing/managers.py", line 774, in _callmethod
2016-08-08T11:33:03.132139440Z     raise convert_to_error(kind, result)
2016-08-08T11:33:03.132883671Z KeyError: 'InterfaceName'

```

If we go to the error line (line 282 in experiment.py), we have the following.

```
276     # Ok we did not get any information within the grace period we give up
277     if not (check_modem_meta(meta_info['modem'], meta_grace) or
278             len(meta_info['gps']) < 1):
279         print "No Metadata or no ip adress on interface: aborting"
280         sys.exit(1)
281 
282     ifname = meta_info['modem']['InterfaceName']
```

**Comment by jonakarl** [comment]
Fixed in commit ef965f6

**Comment by relet** [comment]
Will you let Eneko know, so he can test this?
